Kickstarter-iOS/Views/Controllers/PledgeDisclaimerViewController.swift

```diff
@@ -0,0 +1,175 @@
+        self?.presentHelpWebViewController(with: .trust, presentationStyle: .formSheet)
+      }
+  }
+}
+
+extension PledgeDisclaimerViewController: UITextViewDelegate {
+  func textView(
+    _: UITextView, shouldInteractWith _: NSTextAttachment,
+    in _: NSRange, interaction _: UITextItemInteraction
+  ) -> Bool {
+    return false
+  }
+
+  func textView(
+    _: UITextView, shouldInteractWith _: URL, in _: NSRange,
+    interaction _: UITextItemInteraction
+  ) -> Bool {
+    self.viewModel.inputs.learnMoreTapped()
+    return false
+  }
+}
+
+// MARK: - Styles
+
+private let textViewStyle: TextViewStyle = { (textView: UITextView) -> UITextView in
+  textView
+    |> tappableLinksViewStyle
+    |> \.font .~ UIFont.ksr_caption1()
+    |> \.attributedText .~ attributedLearnMoreText()
+    |> \.textColor .~ UIColor.ksr_text_dark_grey_500
+    |> \.accessibilityTraits .~ [.staticText]
+    |> \.backgroundColor .~ .ksr_grey_400
+}
+
+private func iconImageViewStyle(_ isAccessibilityCategory: Bool) -> (ImageViewStyle) {
+  return { (imageView: UIImageView) in
+    let image = Library.image(named: "icon-not-a-store")?.withRenderingMode(.alwaysTemplate)
+
+    return imageView
+      |> \.image .~ image
+      |> \.tintColor .~ .ksr_green_500
+      |> \.contentMode .~ (isAccessibilityCategory ? .top : .center)
+  }
+}
+
+private let leftColumnStackViewStyle: StackViewStyle = { stackView in
+  stackView
+    |> \.layoutMargins .~ UIEdgeInsets(topBottom: Styles.gridHalf(3))
+    |> \.isLayoutMarginsRelativeArrangement .~ true
+}
+
+private let rootStackViewStyle: StackViewStyle = { stackView in
+  stackView
+    |> \.layoutMargins .~ UIEdgeInsets(topBottom: Styles.grid(2), leftRight: Styles.grid(3))
+    |> \.spacing .~ Styles.grid(2)
+    |> \.isLayoutMarginsRelativeArrangement .~ true
+}
+
+// MARK: - Functions
+
+private func attributedLearnMoreText() -> NSAttributedString? {
+  guard let trustLink = HelpType.trust.url(
+    withBaseUrl: AppEnvironment.current.apiService.serverConfig.webBaseUrl
+  )?.absoluteString else { return nil }
+
+  let paragraphStyle = NSMutableParagraphStyle()
+  paragraphStyle.lineSpacing = 2
+
+  let attributedLine1String = Strings.Kickstarter_is_not_a_store()
+    .attributed(
+      with: UIFont.ksr_body(size: 13),
+      foregroundColor: .ksr_text_dark_grey_500,
+      attributes: [.paragraphStyle: paragraphStyle],
+      bolding: [Strings.Kickstarter_is_not_a_store()]
+    )
+
+  let line2String = Strings.Its_a_way_to_bring_creative_projects_to_life_Learn_more_about_accountability(
+    trust_link: trustLink
+  )
+
+  guard let attributedLine2String = try? NSMutableAttributedString(
+    data: Data(line2String.utf8),
+    options: [
+      .documentType: NSAttributedString.DocumentType.html,
+      .characterEncoding: String.Encoding.utf8.rawValue
+    ],
+    documentAttributes: nil
+  ) else { return nil }
+
+  let attributes: String.Attributes = [
+    .font: UIFont.ksr_body(size: 13),
```

Comment:
Size overrides... 👎

Reply:
Hmm trying to recall why we left this one in there 🤔